Foxtrot/Tests/CRASanitizeTest.cs

```diff
@@ -16,55 +16,44 @@
 using System.Text;
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.IO;
 using System.Xml.Linq;
+using Xunit;
+using Xunit.Abstractions;
 
 namespace Tests
 {
-  [TestClass]
   public partial class Test
   {
-    private TestContext testContextInstance;
+    private readonly ITestOutputHelper _testOutputHelper;
```

Comment:
We could, but it probably wouldn't resolve to [Xunit.Abstractions.ITestOutputHelper](https://github.com/xunit/xunit/blob/master/src/abstractions/xunit.abstractions/TestCases/ITestOutputHelper.cs). :wink: